blas/src/main/java/dev/ludovic/netlib/blas/InstanceBuilder.java

```diff
@@ -52,7 +61,7 @@ private static NativeBLAS initializeNative() {
     try {
       return JNIBLAS.getInstance();
     } catch (Throwable t) {
-      log.log(Level.FINE, "Failed to load implementation from:" + JNIBLAS.class.getName(), t);
+      log.log(Level.FINE, "Failed to load implementation from: " + JNIBLAS.class.getName(), t);
```

Comment:
While you're at it, could you please also fix the indentation at https://github.com/luhenry/netlib/pull/28/files#diff-fc176c1380a1ab2198c7cce2137f928e21c9270cd4126cea19fc4a47c162ee3aR64. Thanks!

Reply:
sorry for missing that, fixed.